starlette/applications.py

```diff
@@ -61,6 +61,7 @@ def __init__(
         on_startup: typing.Optional[typing.Sequence[typing.Callable]] = None,
         on_shutdown: typing.Optional[typing.Sequence[typing.Callable]] = None,
         lifespan: typing.Optional[Lifespan["AppType"]] = None,
+        request_max_size: int = 2621440,  # 2.5mb
```

Comment:
Like in Django

Reply:
https://github.com/django/django/blob/main/django/conf/global_settings.py#L310

Reply:
What does flask do?

Reply:
@Kludex Flask depends on the configuration option or on 'content-length' header. Otherwise, reads nothing.
https://flask.palletsprojects.com/en/2.3.x/config/#MAX_CONTENT_LENGTH
https://flask.palletsprojects.com/en/2.3.x/patterns/fileuploads/#improving-uploads